scripts/convert_wav_to_g711wav.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020 NVIDIA Corporation
+#
+# USAGE:
+# python convert_wav_to_g711wav.py \
+#   --data_dir=<dir with .wav files> \
+#   --dest_dir=<destination dir root>
+#
+# Converts all wav audio files to PCM u-law wav files (8kHz, 8-bit).
+# Requires sox to be installed.
+import argparse
+import concurrent.futures
+import glob
+import logging
+import os
+import subprocess
+
+from tqdm import tqdm
+
+parser = argparse.ArgumentParser(description='Convert wav audio to pcm mulaw wav')
+parser.add_argument(
+    "--data_dir", default=None, type=str, required=True, help="The path to the input directory with .wav files.",
+)
+parser.add_argument(
+    "--dest_dir", default=None, type=str, required=True, help="Path to the destination directory.",
+)
+args = parser.parse_args()
+
+def __convert_audio(in_path, out_path):
+    """
+    Helper function that's called per thread, converts wav to G.711 wav.
+    Args:
+        in_path: source wav file to convert
+        out_path: destination for G.711 wav file
+    """
+    cmd = ["sox", in_path, "-r", "8000", "-c", "1", "-e", "u-law",out_path]
```

Comment:
This script is based on existing script fisher_audio_to_wav.py. This line is the biggest difference other than comments.